HW7/Vincent/code_component/src/page.js

```diff
@@ -0,0 +1,12 @@
+import Image from "next/image";
+import { DogImg } from "@/components/dog_img";
+import { TodoList } from "@/components/todo_list";
+
+export default function Home() {
+  return (
+    <div>
+      <DogImg></DogImg>
+      <TodoList></TodoList>
```

Comment:
nitpick: self closing recommended here
```suggestion
      <DogImg />
      <TodoList />
```

Reply:
addressed in commit [f267aba](https://github.com/Bean-Vic/Homework-Aug-2025/pull/60/commits/f267aba5a9848541d753b2f5d07ca8ea682c1120)